src/agent/src/process_options_unix.cpp

```diff
@@ -0,0 +1,58 @@
+#include <process_options.hpp>
+
+#include <agent.hpp>
+#include <fmt/format.h>
+#include <fmt/ranges.h>
+#include <logger.hpp>
+#include <unix_daemon.hpp>
+
+#include <iostream>
+#include <vector>
+
+void RestartAgent()
+{
+    StopAgent();
+    StartAgent();
+}
+
+void StartAgent()
+{
+    LogInfo("Starting Wazuh Agent.");
+
+    if (UnixDaemon::IsAlreadyRunning())
+    {
+        LogInfo("Wazuh-Daemon is already running");
+        return;
+    }
+
+    const int ret = UnixDaemon::Daemonize();
+
+    if (ret != 0)
+    {
+        LogError("Wazuh Daemon start failed");
```

Comment:
```suggestion
        LogError("Wazuh-Daemon start failed");
```
To be consistent, though we should improve all the log messages.

Reply:
Done